huaweicloud/data_source_huaweicloud_gaussdb_mysql_instance_test.go

```diff
@@ -65,6 +65,9 @@ resource "huaweicloud_gaussdb_mysql_instance" "test" {
 
 data "huaweicloud_gaussdb_mysql_instance" "test" {
   name = huaweicloud_gaussdb_mysql_instance.test.name
+  depends_on = [
+    huaweicloud_gaussdb_mysql_instance.test.name,
```

Comment:
depends_on must be to a resource or data source, not to an  attribute

Reply:
yes, indeed! have fixed